src/Microsoft.VisualBasic/tests/VB/Dummy.vb

```diff
@@ -6,16 +6,19 @@ Imports Microsoft.VisualBasic
 Imports System
 Imports Xunit
 
-Namespace Microsoft.VisualBasic.Tests
+Namespace Microsoft.VisualBasic.Tests.VB
 
-    Public Class Dummy
+    Public NotInheritable Class DummyTest
 
         <Fact>
-        Public Sub Dummy()
-
-            Dim dateTimeNowBefore As DateTime = DateTime.Now()
-            Dim now As DateTime = DateAndTime.Now()
-            Dim dateTimeNowAfter As DateTime = DateTime.Now()
+        Public Shared Sub Dummy()
+            'While (Not System.Diagnostics.Debugger.IsAttached)
+            '    System.Threading.Thread.Sleep(1000)
+            'End While
```

Comment:
I'm not sure what this file represents, but are any of the changes here necessary?

Reply:
It was a sample by Dan for future implementers until debugger got fixed.

Reply:
The original file has the code, but since I am testing I needed to comment it out, I originally deleted it by mistake. This change restores the file to its original state.